bucket/wsltty.json

```
@@ -11,8 +11,11 @@
         "url": "https://github.com/mintty/wsltty/releases/download/$version/wsltty-$version-install.exe#/dl.7z"
     },
     "installer": {
```

Why not rather go with something like:

```suggestion
     "pre_install": [
         "$cont = Get-Content \"$dir\\install.bat\"",
         "$cont.Insert(2, \"cd '$dir'\")", // This maybe fixed size, so some workaround as  [System.Collections.ArrayList] $cont = $cont OR [..ArrayList] $cont = Get-Content ...
         "Set-Content \"$dir\\install.bat\" ($cont -join \"`r`n\") -Encoding ASCII"
     ]
```

---

Or even something like `cd %~dp0` should work instead of let powershell evaluate $dir

---

Should do the trick without breaking users context

```suggestion
     "pre_install": [
         "[System.Collections.ArrayList] $cont = Get-Content \"$dir\\install.bat\"",
         "$cont.Insert(2, 'pushd \"%~dp0\"')",
         "$cont.Insert(-1, 'popd')",
         "Set-Content \"$dir\\install.bat\" ($cont -join \"`r`n\") -Encoding ASCII"
     ],
     "installer": {

```